Build Issue

> Task :examples:hanksTodoRuby:genStubs
Exception in thread "main" java.lang.NullPointerException
        at sapphire.compiler.StubGenerator.generateStubs(StubGenerator.java:72)
        at sapphire.compiler.StubGenerator.generateStubs(StubGenerator.java:107)
        at sapphire.compiler.StubGenerator.main(StubGenerator.java:126)


FAILURE: Build failed with an exception.


![buildissue_2018-10-26 11-52-12](https://user-images.githubusercontent.com/9107906/47548159-b809f100-d915-11e8-8d19-6a9bcbe0e031.png)

--- a/sapphire/sapphire-core/src/main/java/sapphire/compiler/StubGenerator.java
+++ b/sapphire/sapphire-core/src/main/java/sapphire/compiler/StubGenerator.java
@@ -66,9 +66,12 @@ public class StubGenerator {
         return filename.substring(0, extensionIndex);
     }
 
-    public static void generateStubs(String srcFolder, String packageName, String destFolder) {
+    public static void generateStubs(String srcFolder, String packageName, String destFolder) throws java.io.FileNotFoundException {
         File directory = new File(srcFolder);
         File[] fList = directory.listFiles();
+        if (fList == null) {
+            throw new java.io.FileNotFoundException("Failed to list files in source folder \'" + srcFolder + "\'");
+        }
         for (File file : fList) {
             if (file.isFile() && file.getName().endsWith(".class")) {
                 try {
@@ -119,7 +122,7 @@ public class StubGenerator {
      */
 
     /* TODO: Support for multiple packages for app stubs; right now you must run this for each app package that contains a SapphireObject */
-    public static void main(String args[]) {
+    public static void main(String args[]) throws java.io.FileNotFoundException {
         String src = args[0];
         String pkg = args[1];
         String dst = args[2];

